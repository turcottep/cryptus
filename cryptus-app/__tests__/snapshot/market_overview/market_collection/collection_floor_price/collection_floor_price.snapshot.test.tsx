import React from "react";
import renderer from "react-test-renderer";
import "@testing-library/jest-dom/extend-expect";

import CollectionFloorPrice from "../../../../../components/market/market_collection/collection_floor_price/collection_floor_price";

describe("<CollectionFloorPrice />", () => {
  it("should display properly", async () => {
    const test_fp = 7.79;
    const test_delta = 0.01;
    const test_timestamp = "week";

    const tree = renderer
      .create(
        <CollectionFloorPrice
          floor_price_live={test_fp}
          floor_price_delta={test_delta}
          floor_price_timestamp={test_timestamp}
        />
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
