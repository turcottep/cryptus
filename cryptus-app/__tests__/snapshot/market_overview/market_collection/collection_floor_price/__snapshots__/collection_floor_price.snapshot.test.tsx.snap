// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CollectionFloorPrice /> should display properly 1`] = `
<div>
  <div>
    7.79
     Ξ
    <div>
       ETH
    </div>
  </div>
  <div>
    <span>
      0.01
    </span>
    Ξ (
    <span>
      0.13
    </span>
    %)
  </div>
</div>
`;
